src/Serializer/AbstractItemNormalizer.php

```diff
@@ -640,6 +640,13 @@ protected function getAttributeValue(object $object, string $attribute, string $
             return $this->serializer->normalize($attributeValue, $format, $childContext);
         }
 
+        if ($type && $type->getBuiltinType() === 'array') {
+            $childContext = $this->createChildContext($context, $attribute, $format);
+            unset($childContext['iri'], $childContext['uri_variables']);
```

Comment:
todo: check if we shouldn't remove `operation` also

Reply:
could you explain me what choices lead to `unset()` these contexts?
because we're serializing "non api resource" objects, there would be no need to have the operation available in the context?

Reply:
because it'll re-use the old operation if any when creating an IRI

Reply:
unless I'm misunderstanding something, no IRI should be generated, since we're dealing with non-apiresource objects?

Reply:
Yes they have Skolem IRIs unless disabled by using the `genId: false` attribute.